src/posh-git.psd1

```diff
@@ -50,7 +49,7 @@ CmdletsToExport = @()
 VariablesToExport = @()
 
 # Aliases to export from this module
-AliasesToExport = @('??')
+AliasesToExport = @()
```

Comment:
An empty array helps with overall system performance.

If you omit the entry, PowerShell does more expensive work to see if you meant to export any aliases.